client/me/purchases/concierge-banner/index.jsx

```diff
@@ -42,86 +42,91 @@ class ConciergeBanner extends Component {
 	getBannerContent() {
 		const { bannerType, translate } = this.props;
 
-		let headerText;
-		let mainText;
+		let title;
+		let description;
 		let buttonText;
```

Comment:
Thought for our next iteration: it would be good to work out if we can add some basic icons to the banners to add a bit of colour/pop and context.